x/evm/keeper/migrate.go

```diff
@@ -49,6 +61,102 @@ func removeExternalTokenBurnerCode(ctx sdk.Context, ck chainKeeper) error {
 	return nil
 }
 
+func migrateCommandBatchSignature(ctx sdk.Context, ck chainKeeper, signer types.Signer, multisig types.MultisigKeeper) error {
+	commandBatchID := ck.getLatestSignedCommandBatchID(ctx)
+	for {
+		commandBatchMetadata := ck.getCommandBatchMetadata(ctx, commandBatchID)
+		if commandBatchMetadata.ID == nil {
+			break
```

Comment:
why is this a break, not an error?

Reply:
a check to exit the loop if migrated all command batch. 
won't happen on testnet/mainnet migration, since we only migrate active keys.